std_msgs/Header header   
sensor_msgs/Image camera_image    #TODO: Eventually add other cameras

float32[2] gaze_position 
float32[3] gaze_position_3d

EyeData right_eye
EyeData left_eye

float32[3] acelerometer 
float32[3] gyroscope

